### PR TITLE
Nd accessibility

### LINKED ISSUE #47: Refactor: accessibility
- use lighthouse and/or wave to audit accessibility features and fix HTML tags/etc accordingly

### LINKED ISSUE #95: Accessibility: background and foreground colors do not have enough contrast for colorblindness
h2, h4, filter-btn

### LINKED ISSUE #96: Accessibility: naming in html for screen readers
Names and labels:These are opportunities to improve the semantics of the controls in your application. This may enhance the experience for users of assistive technology, like a screen reader.

### LINKED ISSUE #97: Accessibility: Form elements do not have associated labels
input#search-input

### LINKED ISSUE #98: Accessibility: check for logical tabbing order
- User focus is not accidentally trapped in a region
- Visual order on the page follows DOM order

### LINKED ISSUE #99: Accessibility: where do ARIA tags need to be added?
may coincide with #98

### PR DESCRIPTION
**Pull request by npdarrington**
### What’s this PR do?  
- This PR changes many non-semantic HTML tags into semantic HTML tags for better accessibility.
- Adds aria-* descriptions to specific elements that a user can click or tab over.
- Allows a user to tab through elements they can interact with throughout the website.
- Changes some colors that lighthouse was showing an inconsistent contrast against.
- Raises lighthouse accessibility score to 100% for the website.
- Makes a label visible, that was previously hidden, that was causing the lighthouse score to drop by 10 points.
- Changes the code that is injected through JS to have semantic HTML tags and/or aria labels where applicable.

### Where should the reviewer start?  
- Pull these changes down.
- Run `npm start` to boot up local server and navigate to localhost:8080.

### How should this be manually tested?  
- Check for new color schemes to stay in line with the current website color schema.
- Tab through the website and make sure elements are properly being tabbed through.
- Hit `return` or `enter` when tabbing through the website to ensure all elements are keyboard accessible.

### Any background context you want to provide?  
- The buttons inside the header (search button, my recipes button and my pantry button) has it's outline hidden. However, this makes it really difficult to know, when tabbing through the website, where the keyboard tab currently is as nothing shows up when tabbing over those elements. Removing the outline from scss creates a very large outlined box around the elements that may cause some undesirable graphics that linger on the page. Looking for feedback on how to approach this to make these elements show visual feedback when tabbing without causing undesirable visuals.
- The unhidden label for the search bar is currently in a position that is not desirable. That will be addressed in the next PR while  making the website responsive and mobile friendly.

### What are the relevant tickets?  
- Closes #99
- Closes #98 
- Closes #97 
- Closes #96 
- Closes #95
- Closes #47

### Screenshots (if appropriate)  
- N/A

### Questions: 
- N/A

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,41 +9,41 @@
   </head>
   <body>
     <header>
-      <h1><span>fresh</span>Picks<img src="./images/apple-logo.png" alt="apple icon" class="header-apple-icon"></h1>
-      <form id="search" role="search">
-        <input id="search-input" type="text" autocomplete="off" placeholder="Search Recipes..."></input>
+      <h1><label>fresh</label>Picks<img src="./images/apple-logo.png" alt="apple icon" class="header-apple-icon"></h1>
+      <form id="search" role="search" aria-describedby="This is the search form where you can filter all recipe cards on the website by an ingredient or a recipe name.">
```
Comment (aemcdonald) [inline src/index.html:13]: Excellent description of the element on the aria tag here, very thorough

**Review by JWoltjen — APPROVED** [review]
Thanks for adding all the ARIA tags, Nathan!

**Review by hannakim91 — APPROVED** [review]
Looks good! Just one slight bug with the search button being spread wider because of the text "search"

**Review by aemcdonald — APPROVED** [review]
Excellent additions with the ARIA descriptors, they are very thorough & detailed. Awesome job getting the lighthouse score up to 100

**Pull request merged**